proto/cmp/services/transport/v3/trip_types.proto

```diff
@@ -0,0 +1,151 @@
+syntax = "proto3";
+
+package cmp.services.transport.v3;
+
+import "cmp/types/v1/baggage.proto";
+import "cmp/types/v1/datetime_range.proto";
+import "cmp/types/v1/duration.proto";
+import "cmp/types/v1/measurement.proto";
+import "cmp/types/v2/location.proto";
+import "cmp/types/v2/price.proto";
+import "cmp/types/v2/product_code.proto";
+import "cmp/types/v2/service_fact.proto";
+import "google/protobuf/timestamp.proto";
+
+// This message type represents a one way trip, either travelling or returning.
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/transport/v3/trip_types.proto.dot.xs.svg)
+//
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/transport/v3/trip_types.proto.dot.svg)
+message Trip {
+  // Schedule for the whole trip
+  cmp.services.transport.v3.TripSchedule schedule = 1;
+
+  // Luggage
+  //
+  // Ex: `3`
+  cmp.types.v1.Baggage baggage = 2;
+
+  // Price
+  //
+  // Ex: `Price(net=240, currency=Currency.CURRENCY_EUR)
+  cmp.types.v2.Price price = 3;
```

Comment:
Previously we had the `price` in the `TripSegment`. Which means it was possible to define prices for individual segments. And now it's not. Is that intentional?